tsconfig.json

```diff
@@ -11,6 +11,5 @@
     },
     "exclude": [
```

Comment:
I think we can remove `exclude` totally since webpack is configuring these things 🌻

Reply:
Done :)